src/components/action-sheet/action-sheet.md.scss

```diff
@@ -144,7 +139,17 @@ $action-sheet-md-icon-margin-start:                     0 !default;
 }
 
 .action-sheet-md .action-sheet-group {
-  overflow: hidden;
+  overflow: scroll;
+
+  background: $action-sheet-md-background;
+}
+
+.action-sheet-md .action-sheet-group:first-child {
+  @include padding(.8rem, 0, 0, 0);
```

Comment:
add here: 
```
 flex-shrink: 2;
```

Reply:
+ Maybe make the `.8rem` a variable -
```scss
$action-sheet-first-group-padding-top: .8rem;
$action-sheet-last-group-padding-bottom: $action-sheet-first-group-padding-top;

$action-sheet-md-first-group-padding-top: $action-sheet-first-group-padding-top;
$action-sheet-md-last-group-padding-bottom: $action-sheet-last-group-padding-bottom;

$action-sheet-wp-first-group-padding-top: $action-sheet-first-group-padding-top;
$action-sheet-wp-last-group-padding-bottom: $action-sheet-last-group-padding-bottom;
```
+ It doesn't seem like there is a need to pad it with `0`, a `group` does not have padding anyway. (use `null`)